packages/web-component/README.md

```diff
@@ -71,14 +71,16 @@ NOTE: This package is a part of a monorepo. so if you make changes in a dependen
-| theme        | **"light"** - Light theme</br>**"dark"** - Dark theme</br>**"os"** - Auto select a theme based on the OS theme settings                                                                                                                   | **"light"**   |
-| debug        | **"true"** - Enable debugger</br>**"false"** - Disable debugger                                                                                                                                                                           | **"false"**   |
-| telemetryKey | **String** - Telemetry public key provided by Descope Inc                                                                                                                                                                                 | **""**        |
-| auto-focus   | **"true"** - Automatically focus on the first input of each screen</br>**"false"** - Do not automatically focus on screen's inputs</br>**"skipFirstScreen"** - Automatically focus on the first input of each screen, except first screen | **"true"**    |
-|              |                                                                                                                                                                                                                                           |               |
+| Attribute      | Available options                                                                                                                                                                                                                         | Default value |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| base-url       | Custom Descope base URL                                                                                                                                                                                                                   | **""**        |
+| theme          | **"light"** - Light theme</br>**"dark"** - Dark theme</br>**"os"** - Auto select a theme based on the OS theme settings                                                                                                                   | **"light"**   |
+| debug          | **"true"** - Enable debugger</br>**"false"** - Disable debugger                                                                                                                                                                           | **"false"**   |
+| preview        | **"true"** - Ignore incoming tokens</br>**"false"** - Do not ignore tokens                                                                                                                                                                | **"false"**   |
```

Comment:
if we decide to expose this, I think that this description is a bit inaccurate, I would say something vague like `Run flow in preview mode`, the flow engine will decide the impact of this

Reply:
good idea, probably best to keep it vague for now